controller/src/mantarray_desktop_app/main_process/server.py

```diff
@@ -1104,12 +1100,13 @@ def after_request(response: Response) -> Response:
                 response_json["root_recording_path"]
             )
     msg = "Response to HTTP Request in next log entry: "
-    if response.status_code == 200:
+    if response.status_code == 200 and "system_status" not in rule.rule:
```

Comment:
This check is identical to the one above. This is a good hint that the branches should either be combined or there is superfluous logic in one of them